### ISSUE TITLE
Can't find WoW Proccess even though I changed path and WoW is open.

### ISSUE THREAD

**Issue by mhenrichsen** [raidensan91/WoW_Fishing_Bot#1]
[+] Checking for matching processes..
[+] Finding WoW application instance.
[!] Can't find WoW instance running.
[!] Exiting.

Hey. I'm getting the error listed above. I changed the path so it matches my WoW classic .exe.

Do I need to change anything else?

**Comment by Jetidball** [comment]
yes, WowClassic.exe is what you need to change it to right now this bot only scans for:
`wow_running = wow_hijack.check_process(["World of Warcraft", "WorldOfWarcraft"])`
you need to add , "WowClassic.exe" into the check_process function. Line 19 of main.py
Copy and paste the below and it should at least get past that part
`wow_running = wow_hijack.check_process(["World of Warcraft", "WorldOfWarcraft", "WowClassic.exe"])`

**Comment by Jetidball** [comment]
I can get mine to run but it does not do anything, just opens a window with inverted colors and sits there. Any advice would be great
